assets/js/app.js

```diff
@@ -36,89 +37,114 @@ Hooks.CodeViewer = {
+    this.pending = this.page();
+  },
+};
 
 Hooks.Diff = {
   mounted() {
-    const diff = this.el.innerText
+    const diff = this.el.innerText;
     const diffHtml = diff2html(diff, {
       drawFileList: true,
-      matching: 'lines',
-      outputFormat: 'side-by-side',
-      highlight: true
+      matching: "lines",
+      outputFormat: "side-by-side",
+      highlight: true,
     });
-    document.querySelector('#diff').innerHTML = diffHtml;
-  }
-}
+    document.querySelector("#diff").innerHTML = diffHtml;
+  },
+};
 
 Hooks.Logs = {
   mounted() {
     this.el.scrollTop = this.el.scrollHeight;
-  }
-}
+  },
+};
 
 //metric dashboard hook /metrics/dashboard
 Hooks.network_charts = {
   mounted() {
     var network_metric_obj = metric_config_obj.create_network_live_visuals();
-    this.handleEvent("network_points", ({
-      points
-    }) => {
-      points = metric_config_obj.structure_metric_points(points)
-
-      network_metric_obj = metric_config_obj.update_network_live_visuals(network_metric_obj, points);
+    this.handleEvent("network_points", ({ points }) => {
+      points = metric_config_obj.structure_metric_points(points);
 
+      network_metric_obj = metric_config_obj.update_network_live_visuals(
+        network_metric_obj,
+        points
+      );
     });
-
-  }
-}
-
+  },
+};
 
 Hooks.explorer_charts = {
-
   mounted() {
     var explorer_metric_obj = metric_config_obj.create_explorer_live_visuals();
 
-    this.handleEvent("explorer_stats_points", ({
-      points
-    }) => {
-      points = metric_config_obj.structure_metric_points(points)
-      explorer_metric_obj = metric_config_obj.update_explorer_live_visuals(explorer_metric_obj, points);
+    this.handleEvent("explorer_stats_points", ({ points }) => {
+      points = metric_config_obj.structure_metric_points(points);
+      explorer_metric_obj = metric_config_obj.update_explorer_live_visuals(
+        explorer_metric_obj,
+        points
+      );
     });
-  }
-}
+  },
+};
 
 Hooks.worldmap = {
   mounted() {
+    this.handleEvent("worldmap_init_datas", ({ worldmap_datas }) => {
+      if (worldmap_datas.length > 0) createWorldmap(worldmap_datas);
+    });
 
-    this.handleEvent('worldmap_init_datas', ({ worldmap_datas }) => {
-      if (worldmap_datas.length > 0) createWorldmap(worldmap_datas)
-    })
+    this.handleEvent("worldmap_update_datas", ({ worldmap_datas }) => {
+      if (worldmap_datas.length > 0) updateWorldmap(worldmap_datas);
+    });
+  },
+};
 
-    this.handleEvent('worldmap_update_datas', ({ worldmap_datas }) => {
-      if (worldmap_datas.length > 0) updateWorldmap(worldmap_datas)
-    })
-  }
-}
+Hooks.CopyToClipboard = {
+  mounted() {
+    this.el.addEventListener("click", (e) => {
+      const clipboardId = e.target.id;
+
+      const textToCopy = document.getElementById(clipboardId).innerHTML;
+
+      navigator.clipboard
+        .writeText(textToCopy)
+        .then(() => {
+          alert("Text copied to clipboard");
```

Comment:
Here I would suggest to not pop an alert as user need to click on ok button of it to continue interaction with UI.
Maybe you can change the button text temporarily to informe the content is copied. Something like this : 
```js
e.target.innerText = "Copied !"
setTimeout(() => {
  e.target.innerText = "Copy"
}, 1500)
```

Reply:
You could also use something like: https://bulma.io/documentation/elements/notification/